Build failure on MacOS using podman

```
Error: preparing container d3ed7931ce28a2b3f8dccb090151f120c89435662bf43e646923c88f7c167449 for attach: lsetxattr Adv360-Pro-ZMK/config/adv360_left.keymap: operation not supported
```

The changes in PR #58 are the culprit. After reverting locally, I'm able to build successfully.

For reference: https://github.com/containers/podman-compose/issues/509

> Would you be able to test my PR? I dont have a mac to test on

I have an M2 MBA on Ventura 13.1 and the PR still doesn't work. I ran `make clean` and then `make`. This is my error:

```
STEP 9/9: CMD ["./build.sh"]
COMMIT zmk
--> deed133ce29
Successfully tagged localhost/zmk:latest
deed133ce29b5c4f550a8a7dd2c2279c77b14f51865d30a4f28b72b67b0c5918
/opt/homebrew/bin/podman run --rm -it --name zmk \
                -v /Users/kibner/src/Adv360-Pro-ZMK/firmware:/app/firmware:z \
                -v /Users/kibner/src/Adv360-Pro-ZMK/config:/app/config:ro,z \
                -e TIMESTAMP=20230113202756 \
                zmk
Error: preparing container 24c739ca68cf1ba4724e869dc56491774750e777a4542628153483c298ebfa34 for attach: lsetxattr /Users/kibner/src/Adv360-Pro-ZMK/firmware: operation not supported
make: *** [all] Error 126
```

--- a/Makefile
+++ b/Makefile
@@ -1,13 +1,21 @@
 DOCKER := $(shell { command -v podman || command -v docker; })
 TIMESTAMP := $(shell date -u +"%Y%m%d%H%M%S")
+detected_OS := $(shell uname)  # Classify UNIX OS
+ifeq ($(detected_OS),Darwin) #We only care if it's OS X
+SELINUX1 :=
+SELINUX2 :=
+else
+SELINUX1 := :z
+SELINUX2 := ,z
+endif
 
 .PHONY: all clean
 
 all:
 	$(DOCKER) build --tag zmk --file Dockerfile .
 	$(DOCKER) run --rm -it --name zmk \
-		-v $(PWD)/firmware:/app/firmware:z \
-		-v $(PWD)/config:/app/config:ro,z \
+		-v $(PWD)/firmware:/app/firmware$(SELINUX1) \
+		-v $(PWD)/config:/app/config:ro$(SELINUX2) \
 		-e TIMESTAMP=$(TIMESTAMP) \
 		zmk
 

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 DOCKER := $(shell { command -v podman || command -v docker; })
 TIMESTAMP := $(shell date -u +"%Y%m%d%H%M%S")
 detected_OS := $(shell uname)  # Classify UNIX OS
-ifeq ($(detected_OS),Darwin) #We only care if it's OS X
+ifeq ($(strip $(detected_OS)),Darwin) #We only care if it's OS X
 SELINUX1 :=
 SELINUX2 :=
 else

